lib/zone-spec/fake-async-test.ts

```diff
@@ -86,7 +86,7 @@
     private _microtasks: Function[] = [];
     private _lastError: Error = null;
     private _uncaughtPromiseErrors: {rejection: any}[] =
-        Promise[Zone['__symbol__']('uncaughtPromiseErrors')];
+        Promise[Zone['__symbol__']('uncaughtPromiseErrors')] || [];
```

Comment:
good point. I get an error in my tests where I mock the promise to be able to check the status..

hopefully I find time for this...